### PR TITLE
Added mailfolder, delta support to Mail.ReadBasic

### PR DESCRIPTION
**Pull request by jthake-msft**

### CODE REVIEW

File: concepts/changelog.md [hunk 1]
```diff
@@ -59,6 +59,12 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 |Addition|beta|Added the **sharedSecret** member to the [vpnAuthenticationMethod](/graph/api/resources/intune-deviceconfig-vpnauthenticationmethod?view=graph-rest-beta) enum type|
 |Addition|beta|Added the **windows10Home**, **windows10HomeChina**, **windows10HomeN**, **windows10HomeSingleLanguage**, **windows10Mobile**, **windows10IoTCore** and **windows10IoTCoreCommercial** members to the [windows10EditionType](/graph/api/resources/intune-deviceconfig-windows10editiontype?view=graph-rest-beta) enum type|
 
+### Outlook mail
+
+| **Change type** | **Version**   | **Description**                          |
+| :-------------- | :------------ | :--------------------------------------- |
+| Addition | beta | Added support for [delta](/graph/api/message-delta?view=graph-rest-beta) and [mailfolder](/graph/api/user-list-mailfolders?view=graph-rest-beta) methods for Mail.ReadBasic. |
```
Comment (angelgolfer-ms) [inline concepts/changelog.md]: @Lauragra requested my review; may I suggest this changelog description:
"Added support for the Mail.ReadBasic permission, in list and get APIs for mailFolder, delta query for message, and and delta query for mailFolder."

Please add links to list, get, and each of the 2 delta query functions.

**Comment by jthake-msft** [comment]
@angelgolfer-ms  want to take a look again? Thanks.

**Comment by Lauragra** [comment]
@jthake-msft , what is the merge status of this PR? Thanks.

File: concepts/changelog.md [hunk 2]
```diff
@@ -59,6 +59,12 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 |Addition|beta|Added the **sharedSecret** member to the [vpnAuthenticationMethod](/graph/api/resources/intune-deviceconfig-vpnauthenticationmethod?view=graph-rest-beta) enum type|
 |Addition|beta|Added the **windows10Home**, **windows10HomeChina**, **windows10HomeN**, **windows10HomeSingleLanguage**, **windows10Mobile**, **windows10IoTCore** and **windows10IoTCoreCommercial** members to the [windows10EditionType](/graph/api/resources/intune-deviceconfig-windows10editiontype?view=graph-rest-beta) enum type|
 
+### Outlook mail
+
+| **Change type** | **Version**   | **Description**                          |
+| :-------------- | :------------ | :--------------------------------------- |
+| Addition | beta | Added support for the Mail.ReadBasic permission, in [list](/graph/api/user-list-mailfolders?view=graph-rest-beta) and [get](/graph/api/mailfolder-get?view=graph-rest-beta) APIs for mailFolder, [delta query for message](/graph/api/message-delta?view=graph-rest-beta), and [delta query for mailFolder](/graph/api/message-delta?view=graph-rest-beta). |
```
Comment (angelgolfer-ms) [inline concepts/changelog.md]: @jthake-msft , thanks.
The last link should be going to mailfolder-delta.md, not message-delta.md.
Sorry I overlooked a couple of APIs for mailFolder. Maybe we can break the description up into two sentences. "Added support for the Mail.ReadBasic permission in mailFolder API: list mailfolders, get a mailfolder, list child folders, and list messages in a mailfolder. Also added Mail.ReadBasic support in delta query for message and delta query for mailFolder."
Please add links to mailFolder, and each of the APIs mentioned. Thanks!

File: concepts/changelog.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
-| Addition | beta | Added support for the Mail.ReadBasic permission, in [list](/graph/api/user-list-mailfolders?view=graph-rest-beta) and [get](/graph/api/mailfolder-get?view=graph-rest-beta) APIs for mailFolder, [delta query for message](/graph/api/message-delta?view=graph-rest-beta), and [delta query for mailFolder](/graph/api/mailfolder-delta?view=graph-rest-beta). |
+| Addition | beta | Added support for the Mail.ReadBasic permission in mailFolder API: [list mailfolders](/graph/api/user-list-mailfolders?view=graph-rest-beta), [get a mailfolder](/graph/api/mailfolder-get?view=graph-rest-beta), list child folders, and list messages in a mailfolder. Also added Mail.ReadBasic support in [delta query for message](/graph/api/message-delta?view=graph-rest-beta) and [delta query for mailFolder](/graph/api/mailfolder-delta?view=graph-rest-beta).|
```
Comment (angelgolfer-ms) [inline concepts/changelog.md]: @jthake-msft , please add the relative links for "list child folders" (https://docs.microsoft.com/en-us/graph/api/mailfolder-list-childfolders?view=graph-rest-beta) and "list messages in a mail folder" (https://docs.microsoft.com/en-us/graph/api/mailfolder-list-childfolders?view=graph-rest-beta).
I recommend that in the link text, we use plain English to match the TOC entry points, as opposed to using API names. In that case, please make sure "mail folder" is two separate words.

**Comment by jthake-msft** [comment]
@angelgolfer-ms i've updated on your request. Thank you!

**Pull request merged**